packages/babel-traverse/src/path/modification.js

```diff
@@ -126,7 +126,10 @@ export function insertAfter(nodes) {
     if (Array.isArray(this.container)) {
       return this._containerInsertAfter(nodes);
     } else if (this.isStatementOrBlock()) {
-      if (this.node) nodes.unshift(this.node);
+      // Unshift current node if it's not an empty expression
+      if (this.node && (!this.isExpressionStatement() || this.node.expression != null)) {
```

Comment:
I just tried, it doesn't work. 

In this particular case, `this.node` is of type `ExpressionStatement`. `this.isEmptyStatement()` returns `false` because `ExpressionStatement` isn't `EmptyStatement` and they aren't aliases of one another.

I wondered why `this.node` wasn't `EmptyStatement` in the first place and dug in a bit more. Turns out this line https://github.com/babel/babel/blob/7.0/packages/babel-traverse/src/path/replacement.js#L50 sets `this.container['expression'] = null` and `this.container` is `this.node` in the piece of code in question.